tests/unit/test_ipfs.py

```diff
@@ -15,20 +15,30 @@ def test_cid_parsing(self):
         self.assertEqual(
             "bafybeib5q5w6r7gxbfutjhes24y65mcif7ugm7hmub2vsk4hqueb2yylti",
             str(
-                Cid(
+                Cid.old_schema(
                     "0x3d876de8fcd70969349c92d731eeb0482fe8667ceca075592b8785081d630b9a"
                 )
             ),
         )
         self.assertEqual(
             "bafybeia747cvkwz7tqkp67da3ehrl4nfwena3jnr5cvainmcugzocbmnbq",
             str(
-                Cid(
+                Cid.old_schema(
                     "0x1FE7C5555B3F9C14FF7C60D90F15F1A5B11A0DA5B1E8AA043582A1B2E1058D0C"
                 )
             ),
         )
 
+    def test_new_hash_to_cid(self):
+        self.assertEqual(
+            "bafkrwiek6tumtfzvo6yivqq5c7jtdkw6q3ar5pgfcjdujvrbzkbwl3eueq",
+            str(
+                Cid(
+                    "0x8af4e8c9973577b08ac21d17d331aade86c11ebcc5124744d621ca8365ec9424"
+                )
+            ),
+        )
```

Comment:
@gentrexha - this is a test of the new schema!

And it is the same test as the services project uses: see [here](https://github.com/cowprotocol/services/blob/2db800aa38824e32fb542c9e2387d77ca6349676/crates/app-data-hash/src/lib.rs#L63-L76)

Reply:
Thanks a lot for pointing this out! I must've missed it.

Reply:
It was easy to miss in a sea of other tests being changed to test the old style.